02.lovely-movies/styles/styles.css

```diff
@@ -154,6 +154,7 @@ main.content {
     font-size: 16px;
     cursor: pointer;
     margin-bottom: 1rem;
+    float : right;
```

Comment:
This change is not necessary! It's better to suggest new projects or work on best practices.